README.adoc

```diff
@@ -25,6 +25,9 @@ toc::[]
 * https://pinot.apache.org/[Apache Pinot] - A realtime distributed OLAP datastore.
 * https://www.influxdata.com/[InfluxDB] - Purpose-Built Open Source Time Series Database.
 * https://www.postgresql.org/[Postgres] - The World's Most Advanced Open Source Relational Database.
+* http://cassandra.apache.org/[Apache Cassandra] - Open Source distributed wide column store, NoSQL database.
```

Comment:
Nice! Could you only add them in alphabetical order? Thanks!

Reply:
yes, just reordered on datastores section.